### ISSUE TITLE
Sops needs a Website / Docs site / Logo 

### ISSUE THREAD

**Issue by Gowiem** [mozilla/sops#858]
Big fan of sops -- I use it on all new projects. I'm giving a talk at a lookup DevOps meetup about sops in the coming few days and I realized that this awesome project doesn't have a great online presence outside of this repo and its corresponding `README`. I definitely believe it deserves one and I'd love to see that happen! Just figured I'd drop this thought here for ya'll to mull on. 

Thanks!

**Comment by autrilla** [comment]
I agree! I started working on improving the docs in the `docs` branch, https://github.com/mozilla/sops/tree/docs, but as you can see I never got to finishing it. A logo and website would be pretty cool too.

**Comment by nitrocode** [comment]
I like the logo used by walkingtree when they introduced sops

https://walkingtree.tech/secrets-management-using-mozilla-sops/

![image](https://user-images.githubusercontent.com/7775707/173964983-b172f942-210f-4ac3-b647-f6aee89a8280.png)

But I don't know if this image is creative commons or if it's in use by something else. Works well in my current presentation.
